addons/bus/models/ir_websocket.py

```diff
@@ -9,15 +9,6 @@ class IrWebsocket(models.AbstractModel):
     _name = 'ir.websocket'
     _description = 'websocket message handling'
 
-    def _get_im_status(self, im_status_ids_by_model):
```

Comment:
Note for reviewer: this is only used in mail, and requires mail tools when converting to Store.